VMs/metrics/ubuntu16.04/user-data

```diff
@@ -0,0 +1,54 @@
+#cloud-config
+# Copyright (c) 2018 Intel Corporation
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+groups:
+  - jenkins
+
+users:
+  - name: USERNAME
+    gecos: Jenkins User
+    lock-passwd: false
+    shell: /bin/bash
+    sudo: ALL=(ALL) NOPASSWD:ALL
+    ssh-authorized-keys:
+      - PUBLICKEYCONTENTS
+
+packages:
+ - gawk
+ - bc
+ - curl
+ - git
+ - jq
+ - default-jre
+ - gcc
+ - make
+ - build-essential
+
+runcmd:
+
+ - rm /etc/update-motd.d/10-help-text /etc/update-motd.d/51-cloudguest
+ - rm /etc/update-motd.d/90-updates-available
+ - rm /etc/legal
+
+# Install Go
+# The sourceing of environment is a workaround to get any proxy settings we need
+ - 'bash -c "set -a; source /etc/environment; set +a; curl -L https://dl.google.com/go/go1.10.2.linux-amd64.tar.gz -o /tmp/go1.10.2.linux-amd64.tar.gz"'
```

Comment:
Hard-coded golang version and architecture in this block.

Reply:
I know. We only need it for the initial 'go get' of the test repos, and then the CI scripts install a new version...
Longer term maybe I will swap it out for a 'git clone'. Is there a big problem leaving it here right now?

Reply:
Maybe a comment stating that this is purely for bootstrapping purposes might swing it ;)